pages/product.js

```diff
@@ -0,0 +1,44 @@
+const { I } = inject();
+
+
+module.exports = {
+
+  availableOptionsText: 'Available Options',
+  addToCartButton: { xpath: "//button[@id='button-cart']" },
+  priceText: { xpath: "//div[@class='price']//span[@class='price-new']" },
+  sizePrice: { xpath: "//a[@class='sbSelector' and contains (text(), 'Medium')]" },
+  colorPrice: { xpath: "//a[@class='sbSelector' and contains (text(), 'Green')]" },
+  productSizeSelection: { xpath: "//a[@href='#87' and contains(text(), 'Medium')]" },
+  productColorSelection: { xpath: "//a[@href='#85' and contains (text(), 'Green')]" },
```

Comment:
rename to `greenColorOption`. your locator is strict, not generic for all colors